docs/index.rst

```diff
@@ -15,6 +15,7 @@ The Renku Project Docs
 
    About Renku <introduction/index>
    Getting started <getting_started>
+   Upgrading a Renku Project <upgrading_renku>
```

Comment:
I just noticed that this is at the top level - do we really want it here?

Reply:
yeah :D 

my reasoning was that it's something that people don't think to do; if it's in the top level menu it's more visible (and we can keep iterating through what should go in which menu as we re-organize the topics in User (which I don't think should be nested into user at all))

Reply:
but maybe we should think about putting it into a link on one of the first few pages, not in the ToC?

Reply:
why not put it in the ToC?

Reply:
because it's not a top-level topic?

Reply:
i moved it into the "User" section and added the user section to the top level.